web-client/src/views/Messages/MessagesIndividualInbox.jsx

```diff
@@ -18,19 +19,56 @@ const {
 
 export const MessagesIndividualInbox = connect(
   {
+    caseStatuses: state.formattedMessages.caseStatuses,
```

Comment:
Nit: Can formattedMessages just be pulled in once and then properties accessed off of it?

Reply:
@rachelschneiderman -- I agree, but in the interest of time i'm going to merge this. Perhpas add it to devex?